src/s3b_the_roach_market.tw2

```diff
@@ -344,3 +357,46 @@ I’d found this same box on this same table as a kid, and snatched it up immedi
 Honestly I was kinda hurt to see the box still on the table, but... whatever. Today’s ninjas had no fuckin’ taste.
 
 [[(back)|RM-junkheap-goodshit]]
+
+(set: $junkHeapItemsSeen to it +1)
+
+::RM-junkheap-flashback
+I walked up to the scooter and just took it in for a bit. It wasn't in good shape, nothing in this pile could say that.
+
+It was beaten to hell, with rust gathering all over it. The light had been busted out long ago by some ninja looking for a quick thrill. I felt myself pulled back to the [first time I saw the scooter]<scooterTrigger|.
+
+<div class="flashback">(click: ?scooterTrigger)[$dissolve[It’s a couple years ago and I’m waiting outside of the school trailer. I’m waiting for Anna, as usual.
+
+We’re gonna head out to the far fields – where you can actually see the river. The only lights are a Fed boat doing a routine patrol and the only people dudes who’ve wandered too far on a bad trip.
+
+Leo comes roaring up on this tricked-out, midnight black scooter. It’s got pristine hatchet man decals and a fresh paint job. It’s so new that it seems like it’s sparkling all over.
+
+He pulls up next to me, the engine thundering so loud I have to scream at him. “LOOK AT YOU, HUH? [SOME BIG DEAL NOW]<scooterTrigger2|?”]]
+
+(click: ?scooterTrigger2)[$dissolve[I knew it was what he wanted, but I thought that would make the conversation end sooner. He sidles up even closer and cuts the engine. “Pop took a job for the Honchos. Fixed up a few old vans,” he grinned, “We’re rolling in it this week.”
+
+Before I can even respond, he says, “Wanna live like a queen today?”
+
+I know what he wants. We all know what boys want from now until the end of time. It’s not like I’m naive or that I don’t want some version of love too. It felt like every opportunity to say yes might be my last chance.
+
+So I said yes, more to finding out what was next than to him in particular, and we rode off together into the beginning of our story.
+
+I thought I’d see Anna with a big frown, watching me through the scooter dust. But, when I looked [she wasn’t there]<scooterTrigger3|.]]
+
+(click: ?scooterTrigger3)[$dissolve[When I found her the next day, she just wanted details. I barfed a flood of so much boring detail that it should’ve crushed us both. I started sketching his name on things in idle silences.
+
+By the next week, she was tired of hearing about us. I started practicing alone before shows and moved my baggage over to the new relationship. Took a few looks at the old friendship before I tossed the keys and [closed the door]<scooterTrigger4|.]]
+
+(click: ?scooterTrigger4)[$dissolve[When it ended with Leo three weeks later, I felt busted worse than a snowglobe hit with a baseball bat. It hadn’t taken long for his new ride to get dinged up and lose its shine. Some shithead scraped it for a laugh, later his buddies wanna take a joy ride and they took it into a ditch.
```

Comment:
"Some shithead scraped it for a laugh, later his buddies wanna take a joy ride and they took it into a ditch."

Some issues with verb tense here, plus a comma splice. Suggestion:

"Some shithead scraped it for a laugh; later, his buddies took it for a joyride and rode it straight into a ditch."

Reply:
Fixed